pdr_backend/util/web3_accounts.py

```diff
@@ -0,0 +1,98 @@
+import sys
```

Comment:
if you add a new file, please add its corresponding `test_` file and fill it out, as well.

Reply:
in progress

please note that this PR is in draft, and not ready for review